internal/proto/controller/api/resources/authmethods/v1/auth_method.proto

```diff
@@ -303,3 +308,266 @@ message OidcAuthMethodAuthenticateTokenResponse {
+      that: "Certificates"
+    }
+  ]; // @gotags: `class:"public"`
+
+  // client_certificate is the optional certificate encoded as PEM. It must be
+  // set if an optional client_certificate_key specified
+  google.protobuf.StringValue client_certificate = 160 [
+    json_name = "client_certificate",
+    (custom_options.v1.generate_sdk_option) = true,
+    (custom_options.v1.mask_mapping) = {
+      this: "attributes.client_certificate",
+      that: "ClientCertificate"
+    }
+  ]; // @gotags: `class:"public"`
+
+  // Input only. The client_certificate_key (optional) is the plain-text of the
+  // certificate key data encoded as PEM.
+  google.protobuf.StringValue client_certificate_key = 170 [
+    json_name = "client_certificate_key",
+    (custom_options.v1.generate_sdk_option) = true,
+    (custom_options.v1.mask_mapping) = {
+      this: "attributes.client_certificate_key",
+      that: "ClientCertificateKey"
+    }
+  ]; // @gotags: `class:"secret"`
+
+  // Output only. The HMAC'd value of the client certificate key to indicate
+  // whether the certificate key has changed.
+  string client_certificate_key_hmac = 180 [json_name = "client_certificate_key_hmac"]; // @gotags: `class:"public"`
+
+  // bind_dn (optional) is the distinguished name of entry to bind when
+  // performing user and group search. Example:
+  // cn=vault,ou=Users,dc=example,dc=com
+  google.protobuf.StringValue bind_dn = 190 [
+    json_name = "bind_dn",
+    (custom_options.v1.generate_sdk_option) = true,
+    (custom_options.v1.mask_mapping) = {
+      this: "attributes.bind_dn",
+      that: "BindDn"
+    }
+  ]; // @gotags: `class:"public"`
+
+  // Input only. The bind_password (optional) is the password to use along with
+  // binddn when performing user search.
+  google.protobuf.StringValue bind_password = 200 [
+    json_name = "bind_password",
+    (custom_options.v1.generate_sdk_option) = true,
+    (custom_options.v1.mask_mapping) = {
+      this: "attributes.bind_password",
+      that: "BindPassword"
+    }
+  ]; // @gotags: `class:"secret"`
+
+  // Output only. The HMAC'd value of the bind password to indicate
+  // whether the password has changed.
+  string bind_password_hmac = 210 [json_name = "bind_password_hmac"]; // @gotags: `class:"public"`
+
+  bool use_token_groups = 220 [
+    json_name = "use_token_groups",
+    (custom_options.v1.generate_sdk_option) = true,
+    (custom_options.v1.mask_mapping) = {
+      this: "attributes.use_token_groups"
+      that: "UseTokenGroups"
+    }
+  ]; // @gotags: `class:"public"`
+
+  // account_attribute_maps are optional attribute maps from custom attributes
+  // to the standard attributes of fullname and email.  These maps are
+  // represented as key=value where the key equals the from_attribute and the
+  // value equals the to_attribute.  For example "preferredName=fullName".  All
+  // attribute names are case insensitive.
+  repeated string account_attribute_maps = 113 [
```

Comment:
Tiniest nit: What is this tag number 😛? Looks like it should be 230 to continue the convention.
```suggestion
  repeated string account_attribute_maps = 230 [
```

Reply:
ty!